external-plugins/rehearse/plugin/events_server/events_server_test.go

```diff
@@ -0,0 +1,171 @@
+package events_server
```

Comment:
Golang packages does not containe underscroes they just concatenate `envetsserver`